bpm/bonita-core/bonita-process-instance/bonita-process-instance-impl/src/main/resources/org/bonitasoft/engine/core/process/instance/model/impl/hibernate/process.instance.queries.hbm.xml

```diff
@@ -83,6 +83,7 @@
 		WHERE g.logicalGroup4 = :parentProcessInstanceId
 		AND g.name = :name
 		AND g.terminal = FALSE
+		AND g.hitBys NOT LIKE 'FINISH%'
```

Comment:
mmm even if we have first all parameters "logicalGroup4=", "name=" and "terminal="

I'll check if it's possible to use an existing flag